clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -36,7 +36,9 @@ def simple_graph(title, exprs, yAxes=None, legend=""):
     return Graph(
         title=title,
         # One graph per row.
-        targets=[g.Target(expr=expr, legendFormat=legend) for expr in exprs],
+        targets=[
+            g.Target(expr=expr, legendFormat=legend, intervalFactor=1) for expr in exprs
```

Comment:
I've already set the intervalFactor in a PR that merged yesterday (the one that makes grafana use 5s step). You'll need to rebase this.

Reply:
Done